src/silicone/multiple_infillers/infill_all_required_emissions_for_openscm.py

```diff
@@ -289,7 +289,11 @@ def _perform_crunch_and_check(
     cruncher = type_of_cruncher(df)
     filled = [to_fill]
     for req_var in tqdm.tqdm(required_variables, desc="Filling required variables"):
-        infilled = _infill_variable(cruncher, req_var, leaders, to_fill, **kwargs)
+        try:
+            infilled = _infill_variable(cruncher, req_var, leaders, to_fill, **kwargs)
+        except ValueError as err:
+            warnings.warn(f"Error encountered when infilling {req_var}")
```

Comment:
Nice, raise ... from exc is also a good pattern for this extra context https://www.pythontutorial.net/python-oop/python-raise-from/